ui/src/adapters/api/credentials.ts

```diff
@@ -2,22 +2,20 @@ import axios from "axios";
 import { z } from "zod";
 
 import { Schema, schema } from "src/adapters/api/schemas";
+import { Env } from "src/domain";
 import {
   APIResponse,
   HTTPStatusSuccess,
   ResultCreated,
   ResultOK,
   buildAPIError,
 } from "src/utils/adapters";
-import {
-  API_PASSWORD,
-  API_URL,
-  API_USERNAME,
-  ISSUER_DID,
-  QUERY_SEARCH_PARAM,
-} from "src/utils/constants";
+import { QUERY_SEARCH_PARAM } from "src/utils/constants";
 import { StrictSchema } from "src/utils/types";
 
+const buildAuthorizationHeader = (env: Env) =>
```

Comment:
Since we'll need this for all endpoints, it should probably live in `utils`.

Reply:
Yep, I also thought that but, in what utils module? I have resisted the idea of creating another module in utilities just for this function. Should I?

Reply:
It's a good question. Let's leave it for now, I will get back to this in my PR where we need a cohesive approach to env provisioning.